sass/README.md

```diff
@@ -16,3 +16,61 @@ Notable exceptions:
 * prefer double quotes
 
 [1]: http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
+
+## whitespace
+
+Use space between neighboring nested blocks, but not before or after.
+
+### Good
+
+```scss
+.btn {
+  font-size: 15px;
+  display: inline-block;
+
+  &:hover {
+    box-shadow: 0 5px 7px $box-shadow-color;
+  }
+
+  &:active {
+    box-shadow: 0 1px 3px $box-shadow-color;
+    color: $green-dark;
+  }
+}
+```
+
+### Bad
+
+```scss
+.btn {
+
+  font-size: 15px;
+  display: inline-block;
+
+  &:hover {
+    box-shadow: 0 5px 7px $box-shadow-color;
+  }
+
+  &:active {
+    box-shadow: 0 1px 3px $box-shadow-color;
+    color: $green-dark;
+  }
+
+}
+```
+
+### Bad
```

Comment:
I think the third adds to the goldilocks trifecta and there's not much else on this page. Going to leave unless you feel strongly about removing - then let me know!